helm_deploy/values-preprod.yaml

```diff
@@ -38,6 +38,7 @@ generic-service:
     FEATURE_EVENT_OFFENDER_SENTENCES-INSERTED: false
     FEATURE_EVENT_OFFENDER_SENTENCES-DELETED: false
     FEATURE_EVENT_OFFENDER_SENTENCES-UPDATED: false
+    FEATURE_EVENT_PRISONPERSON_PRISON-OFFENDER-EVENTS_PRISONER_BOOKING_MOVED: false
```

Comment:
Turn off only for prison person in preprod (and prod below). Turned on in dev so I can test height/weight works before moving onto the profile details.